cogs/fun.py

```diff
@@ -302,6 +309,25 @@ async def wiki(self, ctx):
         else:
             em = Embed(title='No page given')
         await ctx.send(embed=em)
+
+
+    #136 different Cat Facts!
+    @commands.command(pass_context=True)
+
+    async def catfact(self,ctx):
+        with open('catfacts.json', 'r') as catfacts_file:
+            catfacts_json = catfacts_file.read()
```

Comment:
Would you be terrible put out if I refactored this to dynamically pull from the repo the json file came from?